lib/StateStoreError.js

```diff
@@ -58,6 +59,7 @@ E('ERROR_BAD_ARGUMENT', '%s')
 E('ERROR_NOT_IMPLEMENTED', 'method `%s` not implemented')
 E('ERROR_BAD_CREDENTIALS', 'cannot access %s, make sure your credentials are valid')
 E('ERROR_PAYLOAD_TOO_LARGE', 'key, value or request payload is too large')
+E('ERROR_DB_RATE_LIMITED', 'db access requests rate limited')
```

Comment:
We can change this to following
`E('ERROR_REQUEST_RATE_TOO_HIGH', 'Request rate too high. Please retry after sometime.')`

Reply:
Sure, this looks much better error code and message to be exposed to the user. I'll make the changes.